src/main/java/com/gs/crdtools/BUILD

```diff
@@ -81,3 +81,25 @@ java_binary(
         "@nryaml",
     ],
 )
+
+java_library(
```

Comment:
I think that we should split this in a different way. Let's have one `java_library` for the runtime dependencies (BaseObject, ApiINformation) and one java_binary for the tool. Compiling the same java files multiple times seems like something that can cause confusion down the road

Reply:
good shout! changing this now